src/extensions/CanvasWidget/CanvasPainter.ts

```diff
@@ -265,6 +265,24 @@ export class CanvasPainter {
         this._context2D.fillText(txt, config.x, config.y);
         this._context2D.restore()
     }
+    drawMarker(center: Vec2, opts: {radius: number, pen?: Pen, brush?: Brush}) {
```

Comment:
Implement drawMarker which is similar to drawEllipse+fillEllipse, except the radius is given in pixels rather than in the coordinate system of the painter. In the future we can pass a shape argument, like circle, square, triangle.

This is needed by plots that show markers.